internal/manifests/build_test.go

```diff
@@ -124,8 +123,7 @@ func TestBuildAll_WithFeatureFlags_EnableServiceMonitors(t *testing.T) {
 	}
 
 	for _, tst := range table {
-		testName := fmt.Sprint(tst.desc)
-		t.Run(testName, func(t *testing.T) {
+		t.Run(tst.desc, func(t *testing.T) {
```

Comment:
Encourage you to use `tst := tst` before `t.Run` since you are using `t.Parallel()` and for-loop. Look here:
https://github.com/golang/go/wiki/CommonMistakes